.changeset/warm-colts-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+"@adobe/spectrum-css-workflow-icons": major
+---
+
+chore: take in latest `@a4u/a4u-s2-icon-global-set-open-source-processed`, which is `v5.3.3`.
```

Comment:
Should we update this to be more descriptive? Maybe link to the same resource you did the PR description?

Reply:
Oh with a Breaking change notice?

Reply:
Ok, I updated the changeset to include the `breaking change` note and to be more explicit. I also copied over the changelog info in the delta between this newest release and the `2.0.0` release and put it inside of a `<details>` in this PR description and then linked to this PR in the changeset.